src/main/java/TurnUseCases/BuildUseCase/BuildBuildings.java

```diff
@@ -81,7 +98,8 @@ public boolean buildHouse (Player player, ColorPropertyTile property){
      */
     public boolean buildHotel (Player player, ColorPropertyTile property){
         if (isBuildable(player, property) && property.getHouses() >= 4){
-            property.hotels += 1;
+            property.addHotel();
+            //need to add a method addHotel in ColorPropertyTile class to add the number of hotel.
```

Comment:
I added the add house and add hotel methods in color property tile so you should be able to implement the remaining build methods.